common/templates/options.php

```diff
@@ -181,3 +193,22 @@
         <?php submit_button(); ?>
     </div>
 </form>
+<script>
+    function triggerManualSync() {
+        <?php $nonce = wp_create_nonce($plugin->getManualSyncNonce());?>
+        jQuery.ajax({
+            type: "post",
+            url: "admin-ajax.php",
+            data: {action: <?= json_encode($plugin->getManualSyncAction()); ?>, _ajax_nonce: '<?php echo $nonce; ?>'},
```

Comment:
Use json_encode instead of echoing directly into a string.  You can also JSON encode the whole data object.

~~~
data: <?= json_encode([
  'action' => $plugin->getManualSyncAction(),
  '_ajax_nonce' => $nonce,
]); ?>